.npmignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
Technically, I believe `node_modules` and `npm-debug.log` should automatically be ignored by `npm`, but it's doing no harm to have them here

Reply:
https://docs.npmjs.com/files/package.json#files